src/sic/ast/data/DataChr.java

```diff
@@ -27,7 +27,51 @@ public String toString() {
     public void parse(Parser parser, boolean allowList) throws AsmError {
         parser.advance('C');
         parser.advance('\'');
-        data = parser.readUntil('\'').getBytes();
+        StringBuilder buf = new StringBuilder();
+        for (char c = parser.advance(); c != '\''; c = parser.advance()) {
+            if (c == '\\') {
+                c = parser.advance();
+                switch (c) {
+                    case '\'':
+                        c = '\'';
```

Comment:
This looks redundant. :)

Reply:
Line 36, since c already has that value. You could also fall through to the next case.